docs/source/algorithms/SumSpectra-v1.rst

```diff
@@ -19,24 +19,24 @@ properties (Log values). The properties (Log) names are:
 **NumAllSpectra**, **NumMaskSpectra** and **NumZeroSpectra**,
 where:
 
-* NumAllSpectra is the number of spectra contributed to the sum
+* NumAllSpectra is the number of spectra contributed to the sum
```

Comment:
Note: as this may not be obvious, here and many of the other changes in the pull request are changing from [non-breaking spaces](https://en.wikipedia.org/wiki/Non-breaking_space) to normal spaces.